src/main/kotlin/earth/terrarium/techarium/datagen/DatagenHandler.kt

```diff
@@ -0,0 +1,29 @@
+package earth.terrarium.techarium.datagen
+
+import earth.terrarium.techarium.datagen.providers.*
+import net.minecraft.data.loot.LootTableProvider
+import net.minecraft.world.level.storage.loot.parameters.LootContextParamSets
+import net.neoforged.bus.api.SubscribeEvent
+import net.neoforged.neoforge.data.event.GatherDataEvent
+
+object DatagenHandler {
```

Comment:
Combine this and `TechariumDatagen`

Reply:
ye i was debating if i should or shouldnt i ended up not doing it, but i really didnt know which would be better. Ended up not doing it because the registries were extracted into their own thing too in common so i thought the file needed to be minimalistic